modules/streaming/src/main/scala/io/circe/streaming/ParsingConfiguration.scala

```diff
@@ -0,0 +1,11 @@
+package io.circe.streaming
+
+import jawn.AsyncParser
+
+case class ParsingConfiguration(parseMode: AsyncParser.Mode)
+
+object ParsingConfiguration {
+
+  implicit val defaultParsingConfiguration: ParsingConfiguration =
```

Comment:
@vkostyukov I'd go with implicitis for the sake of backward compatibility. In other case it'd be required to rename old functions in favour of new logic.